application/clicommands/Icinga2Command.php

```diff
@@ -87,6 +87,7 @@ public function configAction()
 EOT
         ];
 
+        $service = false;
```

Comment:
This must be set inside, before the first nested loop. Each second level loop should start fresh, the variable is used inside the first level loop after all, not after it.